### PR TITLE
Frax Pools Implemented

### PR DESCRIPTION
**Pull request by ug02fast**
https://github.com/saddle-finance/saddle-contract/pull/640/

All the pools from above ^

LMK if you want anything changed esp from naming perspective.

### CODE REVIEW

File: src/constants/index.ts [hunk 1]
```diff
@@ -45,6 +45,13 @@ export const FRAX_3_POOL_NAME = "Frax 3Pool"
 export const TBTC_METAPOOL_V2_BTCV2_V3_NAME = "tBTCv2-BTCv2_v3"
 export const WCUSD_METAPOOL_V2_USDV2_V3_NAME = "wcUSD-USDv2_v3"
 export const SUSD_METAPOOL_V2_USDV2_V3_NAME = "sUSD-USDv2_v3"
+export const FRAX_USDC_POOL_NAME = "Saddle FRAX Pool"
+export const FRAX_USDT_METAPOOL_NAME = "frax-USDT Meta"
+export const FRAX_SUSD_METAPOOL_NAME = "frax-sUSD Meta"
+export const FRAX_ALUSD_METAPOOL_NAME = "frax-alUSD Meta"
+export const ARB_FRAX_USDS_METAPOOL_NAME = "frax-USDs Meta"
+export const FTM_FRAX_USDT_METAPOOL_NAME = "frax-USDT Meta"
+export const FTM_FRAX_ALUSD_METAPOOL_NAME = "frax-alUSD Meta"
```
Comment (ug02fast) [inline src/constants/index.ts:54]: FTM pools seperate due to pool token composition order being diff from Saddle FRAX base pool. Added as new entries to pool array.

File: src/constants/index.ts [hunk 2]
```diff
@@ -1099,6 +1297,22 @@ export const EVMOS_TESTNET_POOL_TOKENS = [DAI, USDC, USDT, UST]
...
+export const FRAX_USDT_POOL_TOKENS = [USDT, ...FRAX_USDC_POOL_TOKENS]
+export const FRAX_SUSD_UNDERLYING_POOL_TOKENS = [SUSD, FRAX_USDC_SWAP_TOKEN]
+export const FRAX_SUSD_POOL_TOKENS = [SUSD, ...FRAX_USDC_POOL_TOKENS]
+export const FRAX_ALUSD_UNDERLYING_POOL_TOKENS = [ALUSD, FRAX_USDC_SWAP_TOKEN]
+export const FRAX_ALUSD_POOL_TOKENS = [ALUSD, ...FRAX_USDC_POOL_TOKENS]
+
+export const FTM_FRAX_USDT_UNDERLYING_POOL_TOKENS = [USDT, FTM_USD_SWAP_TOKEN]
+export const FTM_FRAX_USDT_POOL_TOKENS = [USDT, ...FTM_USD_POOL_TOKENS]
+export const FTM_FRAX_ALUSD_UNDERLYING_POOL_TOKENS = [ALUSD, FTM_USD_SWAP_TOKEN]
+export const FTM_FRAX_ALUSD_POOL_TOKENS = [ALUSD, ...FTM_USD_POOL_TOKENS]
```
Comment (ug02fast) [inline src/constants/index.ts:1311]: FTM seperate due to diff token composition

File: src/constants/index.ts [hunk 3]
```diff
@@ -45,6 +45,13 @@ export const FRAX_3_POOL_NAME = "Frax 3Pool"
 export const TBTC_METAPOOL_V2_BTCV2_V3_NAME = "tBTCv2-BTCv2_v3"
 export const WCUSD_METAPOOL_V2_USDV2_V3_NAME = "wcUSD-USDv2_v3"
 export const SUSD_METAPOOL_V2_USDV2_V3_NAME = "sUSD-USDv2_v3"
+export const FRAX_USDC_POOL_NAME = "Saddle FRAX Pool"
```
Comment (hammeiam) [inline src/constants/index.ts:48]: Could we use the same names that will be in the registry? eg https://github.com/saddle-finance/saddle-contract/pull/636/files#diff-2dd57de105b1d5d16969707bd002957c2a520fb6ec196fb6425f604de2085b81R20 "FRAX-USDC-BP", "FRAXBP-SUSD" etc

**Review by hammeiam — APPROVED** [review]
approving to unblock since I don't see anything incorrect in code, however I don't see some pools in the UI. This should not be landed until all pools appear correctly

Screen Shot 2022-06-28 at 3 33 38 PM

**Pull request merged**